features/index/filters.feature

```diff
@@ -249,3 +249,36 @@ Feature: Index Filtering
     """
     And I press "Filter"
     Then I should not see a sidebar titled "Search status:"
+
+  Scenario: Too many categories to show
+    Given a category named "Astrology" exists
+    Given a category named "Astronomy" exists
+    Given a category named "Navigation" exists
+    Given a post with the title "Star Signs" in category "Astrology" exists
+    Given a post with the title "Constellations" in category "Astronomy" exists
+    Given a post with the title "Compass and Sextant" in category "Navigation" exists
+    And an index configuration of:
+    """
+      ActiveAdmin.register Category
+      ActiveAdmin.register Post do
+        config.namespace.maximum_association_filter_arity = 3
+      end
+    """
+    And I am on the index page for posts
+    Then I should see "CATEGORY" within "#filters_sidebar_section"
+     And I should not see "CATEGORY NAME STARTS WITH" within "#filters_sidebar_section"
+    Given an index configuration of:
+    """
+      ActiveAdmin.register Category
+      ActiveAdmin.register Post do
+        config.namespace.maximum_association_filter_arity = 2
```

Comment:
Should we also test the case where  a `starts_with` box is rendered (maximum arity reached, and no explicity filter method configured)?